app/models/katello/concerns/organization_extensions.rb

```diff
@@ -37,10 +37,6 @@ module OrganizationExtensions
         scoped_search :on => :label, :complete_value => :true
 
         after_create :associate_default_capsule
-
-        validates :name, :uniqueness => true, :presence => true
-        validates_with Validators::KatelloNameFormatValidator, :attributes => :name
-        validates :label, :presence => true
         validates_with Validators::KatelloLabelFormatValidator, :attributes => :label
         validate :unique_name_and_label
```

Comment:
I meant, the unique name part of it since Foreman already covers that -- https://github.com/theforeman/foreman/blob/develop/app/models/taxonomy.rb#L28